virtcontainers/virtiofsd.go

```diff
@@ -125,11 +150,23 @@ func (v *virtiofsd) args() ([]string, error) {
 	}
 
 	args := []string{
+		// Send logs to syslog
+		"--syslog",
+		// foreground operation
 		"-f",
+		// cache mode for virtiofsd
 		"-o", "cache=" + v.cache,
+		// disable posix locking in daemon: bunch of basic posix locks properties are broken
+		// apt-get update is broken if enabled
 		"-o", "no_posix_lock",
+		// shared directory tree
 		"-o", "source=" + v.sourcePath,
-		"-o", "vhost_user_socket=" + v.socketPath,
```

Comment:
Not looked into what `vhost_user_socket` was used for, but why is this removed here ?

Reply:
we have two options to provide the socket path and virtiofsd will create the socket or provide the file descriptor. Because no more stdout check is done by the runtime we require that the hypervisor has access to the socket when it starts. Using the `FD` let us do that without keeping checking until the socket appears.